src/org/robockets/steamworks/drivetrain/Joyride.java

```diff
@@ -21,23 +23,25 @@ public Joyride() {
     }
 
     protected void initialize() {
-    	ToggleDriveMode.isArcade = false;
+    	ToggleDriveMode.isArcade = true;
     }
 
     protected void execute() {  	
     	if(ToggleDriveMode.isArcade) {
     		translate = OI.joystick.getRawAxis(1);
         	rotate = OI.joystick.getRawAxis(4);	
         	
-        	RobotMap.robotDrive.arcadeDrive(translate, rotate);
+        	RobotMap.robotDrive.arcadeDrive(-translate, -rotate);
     	} else {
     		leftStick = OI.attack3Left.getRawAxis(1);
     		rightStick = OI.attack3Right.getRawAxis(1);
-    		leftStick /= 2;
-    		rightStick /= 2;
+    		leftStick /= -2;
+    		rightStick /= -2;
     		leftStick *= 1 - OI.attack3Left.getRawAxis(2);
     		rightStick *= 1 - OI.attack3Right.getRawAxis(2);
 
+    		SmartDashboard.putNumber("LeftStickVal", leftStick);
```

Comment:
They're nice to have for debugging. Which I suspect we'll do at least once since our joysticks are trash.